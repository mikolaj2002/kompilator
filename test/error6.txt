# Error6. Redeclaration of variable a in line 5

PROCEDURE pa(T a, b) IS
 c[5], a
IN
  d:=b;
  a[d]:=c[d]+a[b];
END

PROGRAM IS
 a[10],b
IN
  READ b;
  pa(a,b);
  WRITE a[b];
END

