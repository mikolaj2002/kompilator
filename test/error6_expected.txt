[COMPILER] Redeclaration of variable a in line 5
